src/managing-workflow/tuning-component-settings.md

```diff
@@ -0,0 +1,230 @@
+	OPEN_TSDB_DEBUG
+	RIEMANN_URL
+	RIEMANN_TRANSPORT
+	ENABLE_ZFS
+	DISK_MOUNT_POINTS
+	DISKIO_DEVICES
+	DISKIO_SKIP_SERIAL_NUMBER
+	NET_INTERFACES
+	APACHE_URLS
+	DISQUE_SERVERS
+	DOCKER_ENDPOINT
+	DOCKER_CONTAINER_NAMES
+	ELASTIC_SEARCH_SERVERS
+	ELASTIC_SEARCH_LOCAL
+	ELASTIC_SEARCH_CLUSTER_HEALTH
+	HAPROXY_SERVERS
+	HTTP_JSON_NAME
+	HTTP_JSON_SERVERS
+	HTTP_JSON_METHOD
+	HTTP_JSON_TAG_KEYS
+	HTTP_JSON_EVENT_TYPE
+	HTTP_JSON_THRESHOLD
+	ENABLE_INFLUXDB_INPUT
+	INFLUXDB_URLS
+	MEMCACHED_SERVERS
+	MONGODB_SERVERS
+	MYSQL_SERVERS
+	NGINX_URLS
+	NSQ_ENDPOINTS
+	POSTGRESQL_ADDRESS
+	POSTGRESQL_DATABASES
+	PROMETHEUS_URLS
+	PROMETHEUS_INSECURE_SKIP_VERIFY
+	PROMETHEUS_BEARER_TOKEN
+	RABBITMQ_URL
+	RABBITMQ_NAME
+	RABBITMQ_USERNAME
+	RABBITMQ_PASSWORD
+	RABBITMQ_NODES
+	REDIS_SERVERS
+	RETHINKDB_SERVERS
+	ZOOKEEPER_SERVERS
+	KAFKA_CONSUMER_TOPICS
+	KAFKA_CONSUMER_ZOOKEEPER_PEERS
+	KAFKA_CONSUMER_GROUP
+	KAFKA_CONSUMER_POINT_BUFFER
+	KAFKA_CONSUMER_OFFSET
+	STATSD_SERVICE_ADDRESS
+	STATSD_DELETE_GAUGES
+	STATSD_DELETE_COUNTERS
+	STATSD_DELETE_SETS
+	STATSD_DELETE_TIMINGS
+	STATSD_PERCENTILES
+	STATSD_CONVERT_NAMES
+	STATSD_TEMPLATES
+	STATSD_ALLOWED_PENDING_MESSAGES
+	STATSD_PERCENTILE_LIMIT
+	STATSD_UDP_PACKET_SIZE
+
+## Customizing the Registry
+
+The following environment variables are tunable for the [Registry][] component:
+
+Setting           | Description
+----------------- | ---------------------------------
+
+## Customizing the Router
+
+The following environment variables are tunable for the [Router][] component:
+
+Setting           | Description
+----------------- | ---------------------------------
+POD_NAMESPACE     | The pod namespace the router resides (default: "default")
+
+## Customizing Workflow Manager
+
+The following environment variables are tunable for [Workflow Manager][]:
+
+Setting                            | Description
+---------------------------------- | ---------------------------------
+CHECK_VERSIONS    | Enables the external version check at <https://versions.deis.com/> (default: "true")
+POLL_INTERVAL_SEC | The polling interval, in seconds (default: 43200, or 12 hours)
+VERSIONS_API_URL  | The versions API URL (default: "https://versions.deis.com")
+DOCTOR_API_URL    | The doctor API URL (default: "https://doctor.deis.com")
+API_VERSION       | The major version number for this Workflow release (default: "v2")
+
+
+[builder]: ../understanding-workflow/components.md#builder
+[controller]: ../understanding-workflow/components.md#controller
+[database]: ../understanding-workflow/components.md#database
+[gunicorn]: http://gunicorn.org/
+[logger]: ../understanding-workflow/components.md##logger-fluentd-logger
+[minio]: ../understanding-workflow/components.md#minio
+[pull-policy]: http://kubernetes.io/docs/user-guide/images/
+[registry]: ../understanding-workflow/components.md#registry
+[router]: ../understanding-workflow/components.md#router
+[workflow manager]: ../understanding-workflow/components.md#workflow-manager
```

Comment:
Note that this link is broken, but it will go to the document until #252 is implemented.